ios/RNMagicScriptHostApplication/sources/ViewController.swift

```diff
@@ -34,9 +34,10 @@ class ViewController: UIViewController {
         super.viewDidLoad()
         setupScene()
         
-        //        setupPrismWithModels()
-        //        setupPrismForHitTest()
-        setupPrismForDialog()
+//        setupPrismWithModels()
+//        setupPrismForHitTest()
+//        setupPrismForDialog()
```

Comment:
why are you keeping commented out code?

Reply:
This project/code is for debug/testing purposes (for developers). It does not affect anything related to magic-script. It just builds sample scenes from swift code (not from JSX). It is also used for Unit tests project (but with an empty scene).